scss/components/_utils.scss

```diff
@@ -2,3 +2,25 @@
   position: relative;
   top: -1px;
 }
+
+.civicase__3-dot-menu.btn-group {
+  .btn {
+    border: 0;
+    height: 18px;
+    margin-left: 6px;
+    padding: 0;
+    top: 0;
+    width: 18px;
+
+    &.dropdown-toggle {
+      background: transparent;
+      box-shadow: none;
+    }
+  }
+
+  > .dropdown-menu {
+    left: 50% !important; // important used to override incorrect inline styles
```

Comment:
![2018-09-19 at 2 12 pm](https://user-images.githubusercontent.com/5058867/45741811-2ae8c380-bc16-11e8-8d84-69de4738cb57.png)

I think the alignment is wrong. It should be right aligned. So
![2018-09-19 at 2 13 pm](https://user-images.githubusercontent.com/5058867/45741859-42c04780-bc16-11e8-8d68-152bc90743f0.png)

`transform: translateX(-100%);`